src/ferc_xbrl_extractor/cli.py

```diff
@@ -0,0 +1,135 @@
+"""A command line interface (CLI) to the xbrl extractor."""
+import argparse
+import logging
+import sys
+from pathlib import Path
+
+import coloredlogs
+from sqlalchemy import create_engine
+
+from ferc_xbrl_extractor import helpers, xbrl
+from ferc_xbrl_extractor.instance import Instance
+
+DEFAULT_TAXONOMY = "https://eCollection.ferc.gov/taxonomy/form1/2022-01-01/form/form1/form-1_2022-01-01.xsd"
```

Comment:
So a taxonomy is the schema of an xbrl filing?

Reply:
Yeah it provides both metadata and defines the structure